src/globals/scss/styles.scss

```diff
@@ -6,6 +6,7 @@ $css--font-face: false !default;
 $css--helpers: true !default;
 $css--body: true !default;
 $css--reset: false !default;
+$css--typography: true !default;
```

Comment:
`$css--typography: true` will output CSS from `_css--typography.scss` and this is default.